src/main/java/Duke.java

```diff
@@ -11,22 +13,22 @@ public static void main(String[] args) {
     private static void readAndRespond() {
         Scanner sc = new Scanner(System.in);
         String input = sc.nextLine();
-        Task[] listOfTasks = new Task[101]; // 1-base indexing
+        Task[] listOfTasks = new Task[MAX_TASKS]; // 1-base indexing
         while (!input.equals("bye")) {
```

Comment:
I like how all the `if` and `else if` conditional statements in this `while` loop are clearly named and gives readers a good idea of what each statement checks and what the results of each check are. 👍